keyboards/smart/smart80/info.json

```diff
@@ -0,0 +1,304 @@
+{
+    "keyboard_name": "Smart80",
+    "usb": {
+        "pid": "0x5044",
+        "device_version": "0.0.1"
+    },
+
+    "bootmagic": {
+        "matrix": [1, 1]
+    },
```

Comment:
>     "bootloader_instructions": "Press ESC(ROW0COL0), Plug USB. ",

That doesnt match up, either the comment is wrong or the bootmagic config needs to go

```suggestion
```